Library/Formula/libtelnet.rb

```diff
@@ -0,0 +1,39 @@
+require "formula"
+
+class Libtelnet < Formula
+  homepage "https://github.com/seanmiddleditch/libtelnet"
+  url "http://github.com/seanmiddleditch/libtelnet/tarball/0.21"
```

Comment:
Please use the ` https://github.com/seanmiddleditch/libtelnet/archive/0.21.tar.gz ` link. Same link, different formats, but the archive format is the one we usually go with.